fix(content): "Early south missions" adjustments

bene-dictator opened the pull request:
**Bug fix**
This PR adjusts the wording and mission syntax for some of the "Early south missions" content in #9097, and fixes a `branch` node that referred to a missing mission that was renamed.

Amazinite commented on data/human/human missions.txt:
```diff
@@ -6303,9 +6303,9 @@ mission "Blind Man from Martini"
 			`	With that, he grabs his cane and his luggage, and follows a hospital attendant toward his daughter's room, leaving you in the reception.`
 
 mission "Care Package to South 1"
-	minor
```
But that was right. :(

bene-dictator replied:
It was? I better update my style guide...

bene-dictator commented:
I've removed those changes moving `minor`, but I've also noticed that "Stranded In Paradise" could have a better mission name which would fit with other missions, so I've changed that too. @danaris might want to look at that change?

alexrovw commented on data/human/human missions.txt:
```diff
@@ -6583,8 +6583,8 @@ mission "Stranded In Paradise 1"
-			`	"Oh! You may be the answer to our prayers, dearie!" She and the others have now gotten themselves a bit more organized, and you can tell that they must be a family. Confirming this, the woman says, "We're the Burnbecks; I'm Shirley, and this is my husband Darren." She gestures towards the other, younger woman, then the child. "These are our kids, Erin and Leigh" Leigh's features are softer t ...
-			`	Her husband cuts in, "Anyway, the short of it is that we really need to get home, and right now, none of the regular services are running from here to <destination> because of the war. Could you possibly take the four of us back home? We...can't pay a lot, right now, but we can pay."`
+			`	"Oh! You may be the answer to our prayers, dearie!" She and the others have now gotten themselves a bit more organized, and you can tell that they must be a family. Confirming this, the woman says, "We're the Burnbecks; I'm Shirley, and this is my husband Darren." She gestures towards the other, younger woman, then the child. "These are our kids, Erin and Leigh." Leigh's features are softer  ...
+			`	Her husband cuts in, "Anyway, the short of it is that we really need to get home, and right now, none of the regular services are running from here to <planet> because of the likelihood of war. Could you possibly take the four of us back home? We... can't pay a lot, right now, but we can pay."`
```
I actually asked for this change, because this mission fires after the terrorist attacks but before a single shot is fired. The Burnbecks are up-to-date enough on the news that they know about the attacks; there's no reason to believe they wouldn't be up-to-date on whether or not _war were declared_.

bene-dictator replied:
"likelihood" and "possibility" are synonyms to me.

alexrovw replied:
> According to other places in the data files, the war officially "begins" on July 4, 3014 as a cold war, or hasn't actually begun yet by the time of this mission. It's ambiguous, and I feel like that ambiguity should be reflected in this conversation.

I think that event is probably misnamed, and that is likely the cause of the issue in the first place. When I brought this up on Discord, I initially suggested changing the trigger from `event "war begins"` to `event "start of hostilities"` while keeping the mission text the same.

War doesn't really "_officially_" begin either until it's declared or until hostilities begin. As a historical example, the US Civil War didn't begin until the first shots were fired at Fort Sumter on April 12, 1861. By that point, many states had seceded from the Union; some had even seceded in the previous year.

Amazinite replied:
The event won't be renamed for save file compatibility reasons. If we were to rename the event, we'd be retriggering the event for old save files that may have already received it, which can mess things up. So while the event name is "war begins", that's just when the bombing happens. The actual start of the war is whenever the Navy takes Kornephoros.

> "likelihood" and "possibility" are synonyms to me.

In this context, I actually read "likelihood" as stronger than "possibility." "Possibility" feels more like "it could or could not happen," while "likelihood" feels more like "there is a high chance that it will happen." And in that sense, I think it makes sense for this character to say "likelihood."

alexrovw replied:
> The event won't be renamed for save file compatibility reasons.
Just to clarify, I wasn't trying to suggest that we change the event name, only that it's a misnomer :P Thank you for confirming everything!

danaris replied:
Since a primary purpose of this mission is to guide new players to the South to pick up the Free Worlds storyline, changing the trigger condition to an event that only takes place _after_ the player gets down there would defeat the purpose.

The specific reason for the Burnbecks' stranding is less important than the fact that they have been stranded; since it appears I slightly misunderstood the sequence of events when I originally wrote this, the better option is to reword the mission text to have them stranded due to the bombings and the panic that followed.
